app/serializers/assignment_serializer.rb

```
@@ -1,3 +1,17 @@
 class AssignmentSerializer < ActiveModel::Serializer
-     attributes :id, :name, :max_team_size, :course_id
+     attributes :id, :name, :max_team_size, :course_id, :has_role_based_review, :assignment_duties
+
+     def has_role_based_review
```

RuboCop (1.86.1)

[convention] Use 2 (not 5) spaces for indentation.

(Layout/IndentationWidth)